app/views/admin/notifications/_notification.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if notification.present? %>
+  <span class="pl-2 d-none d-md-block notification-item">
+    <i class="fa fa-circle text-<%= notification[:level] %>"></i> <span class="text-light"><%= notification[:body] %></span> <button class="btn btn-dark btn-sm">Dismiss</button>
```

Comment:
It should display properly if there is a link or not.

Currently when there is a link it looks like

![image](https://user-images.githubusercontent.com/8007722/72945649-74857900-3d4a-11ea-8e2e-1163ed7e2b37.png)